backend/capellacollab/tools/models.py

```diff
@@ -301,6 +302,13 @@ class ToolSessionConfiguration(core_pydantic.BaseModel):
             description="Configuration for persistent workspaces.",
         )
     )
+    supported_types: list[project_models.ProjectType] = pydantic.Field(
```

Comment:
```suggestion
    supported_project_types: list[project_models.ProjectType] = pydantic.Field(
```

Not clear which types we talk about otherwise.